package.json

```diff
@@ -108,21 +107,28 @@
     "eslint-plugin-simple-import-sort": "8.0.0",
     "gacp": "3.0.2",
     "husky": "8.0.1",
-    "jest": "29.1.2",
+    "jest": "^26.6.3",
     "lint-staged": "13.0.3",
     "pinst": "3.0.0",
     "prettier": "2.7.1",
     "react": "18.1.0",
-    "react-dom": "18.1.0",
-    "react-native": "0.70.0",
+    "react-dom": "^18.1.0",
+    "react-native": "^0.70.0",
     "react-native-builder-bob": "0.18.3",
+    "react-native-gesture-handler": "^2.6.0",
+    "react-native-reanimated": "^2.10.0",
+    "react-native-svg": "^12.3.0",
     "release-it": "15.1.4",
     "sort-package-json": "2.0.0",
     "typescript": "4.8.3"
   },
   "peerDependencies": {
-    "react": "*",
-    "react-native": "*"
+    "react": "16.9.0 || 16.11.0 || 16.13.1 || 17.0.1 || 17.0.2 || 18.0.0 || 18.1.0",
```

Comment:
@crstnmac Shall we narrow these dependencies to the latest, please?

Reply:
Okay